tests/integration/python/stack_truncate/main_dir_specified/bar/requirements.txt

```diff
@@ -0,0 +1 @@
+pulumi
```

Comment:
You might not need requirements.txt everywhere; this one is misplaced and ignored but the test still works I think.

Reply:
surprisingly, pulumi didn't recognize `requirements.txt` in the base directory for that example and crashed.

Reply:
Huh, weird. OK then.